domain-mapping/inc/redirect.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+defined( 'ABSPATH' ) or die();
+
+/**
+ * The function `rest_get_url_prefix()` is not available at this point in the
+ * load process. Therefore we must substitute it with a close approximate of
+ * what the function does.
+ *
+ * There is a side-effect of this. Basically if a site is to be setup with a
+ * different prefix, in order for this to work, the `add_filter()` call would
+ * need to be done in an Must-Use Plugin.
+ */
+$rest_url_prefix = '/' . trim( apply_filters( 'rest_url_prefix', 'wp-json' ), '/' ) . '/';
+
+/**
+ * Stop the execution of PHP code if any of the following conditions are true.
+ * The constants noted below are all available and prior to the loading of the
+ * sunrise.php file.
+ */
+if (
+        /**
+         * Do not attempt to redirect during the CLI command.
+         */
+        defined( 'WP_CLI' ) && WP_CLI
```

Comment:
each of these lines, where multiple conditionals are evaluated together, should be wrapped in parentheses to avoid it being executed in an order one might not expect.